src/types/example.ts

```diff
@@ -0,0 +1,21 @@
+export type Example = {
+  id: string;
+  igbo?: string;
+  english?: string;
+  meaning?: string;
+  nsibidi?: string;
+  nsibidiCharacters: string[];
+  associatedWords: string[];
+  associatedDefinitionsSchemas: string[];
+  pronunciations: Pronounciation[];
+  updatedAt: Date;
+};
+
+export type Pronounciation = {
```

Comment:
```suggestion
export type Pronunciation = {
```

also nit: i don't think this type needs to be exported since it's not being used

Reply:
@ijemmao 
Refactored, Thanks!